demo-apps/cf-android/app/src/main/java/org/eclipse/californium/examples/ConfigureDtls.java

```diff
@@ -0,0 +1,86 @@
+/*******************************************************************************
+ * Copyright (c) 2015 Institute for Pervasive Computing, ETH Zurich and others.
```

Comment:
The Copyright (c) refelct the year and the authors  organization. For private persons it's just their name.
So I guess, in your case its':

* Copyright (c) 2018 Bala Vikram Reddy Gopu and others

(hope I put your names in the right order :-).

Reply:
I thought it is obligatory for every class, now I will change it.